### PR TITLE
Package: zed.2.0.2

### PR DESCRIPTION
**Pull request by kandu**
* Zed\_utf8: fix an offset-stepping bug in function `unsafe_extract_prev`

### CODE REVIEW

File: packages/zed/zed.2.0.2/opam [hunk 1]
```diff
@@ -0,0 +1,33 @@
+opam-version: "2.0"
+maintainer: "opam-devel@lists.ocaml.org"
+authors: ["Jérémie Dimino"]
+homepage: "https://github.com/ocaml-community/zed"
+bug-reports: "https://github.com/ocaml-community/zed/issues"
+dev-repo: "git://github.com/ocaml-community/zed.git"
+license: "BSD3"
+depends: [
+  "ocaml" {>= "4.02.3"}
+  "dune" {build & >= "1.0.0"}
```
Comment (kit-ty-kate) [inline packages/zed/zed.2.0.2/opam]:
```suggestion
  "dune" {build & >= "1.1.0"}
```
Your `dune-package` file indicates that this package requires dune >= 1.1

File: packages/zed/zed.2.0.2/opam [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+  "base-bytes"
+  "camomile" {>= "1.0.1"}
+  "react"
+  "charInfo_width" {>= "1.1.0" & < "2.0~"}
+]
+build: [
+  ["dune" "subst"] {pinned}
```
Comment (kit-ty-kate) [inline packages/zed/zed.2.0.2/opam]:
```suggestion
```
You don't seem to use dune substitutions anywhere. Is there a reason why you are using `dune subst` here?

Comment (kandu) [reply packages/zed/zed.2.0.2/opam]: no, this is a good time to clean up some old useless code.
Thank you.

**Comment by kit-ty-kate** [comment]
Thanks!

**Pull request merged**